src/app/directives/raml-field.tpl.html

```diff
@@ -9,7 +9,7 @@
   </span>
 
   <select id="select_{{param.id}}" ng-if="isEnum(param)" name="param.id" class="raml-console-sidebar-input" ng-model="model[0]" style="margin-bottom: 0;" ng-change="onChange()">
-   <option ng-repeat="enum in unique(param.enum)" value="{{enum}}">{{enum}}</option>
+   <option ng-repeat="enum in unique(param.enum)" value="{{enum}}" ng-selected="{{param.example === enum}}">{{enum}}</option>
```

Comment:
Default is the value the server must use when the client sends no value (see https://github.com/raml-org/raml-spec/blob/master/raml-0.8.md#default):
> The default attribute specifies the default value to use for the property if the property is omitted or its value is not specified. This SHOULD NOT be interpreted as a requirement for the client to send the default attribute's value if there is no other value to send. Instead, the default attribute's value is the value the server uses if the client does not send a value.